.gitignore

```diff
@@ -48,3 +48,4 @@ app.*.map.json
 
 # Credentials
 lib/.mapbox_credentials.dart
```

Comment:
we really won't need the .mapbox_credentials file anymore (since we should use an env file instead now)